doc/cephadm/monitoring.rst

```diff
@@ -70,3 +70,10 @@ cluster.
 * To enable the dashboard's prometheus-based alerting, see :ref:`dashboard-alerting`.
 
 * To enable dashboard integration with Grafana, see :ref:`dashboard-grafana`.
+
+Enabling the RBD-Image monitoring
+---------------------------------
+
+Currently due to performance reasons the monitoring of rbd images is per default disabled. For more information please
```

Comment:
maybe: Due to performance reasons, monitoring of RBD images is disabled by default.